src/core/agent/services/connectionService.ts

```diff
@@ -312,8 +322,20 @@ class ConnectionService extends AgentService {
       this.props.signifyClient,
       await this.props.signifyClient.oobis().resolve(url, alias)
     );
-    if (!operation.done) {
-      throw new Error(ConnectionService.FAILED_TO_RESOLVE_OOBI);
+    if (!operation.done && !isMultisigRelated) {
+      if (/\/oobi\//.test(url)) {
+        const connectionId = new URL(url).pathname.split("/")[2];
+        const pendingOperation = await this.operationPendingStorage.save({
+          id: operation.name,
+          metadata: { connectionId },
+          recordType: OperationPendingRecordType.Oobi,
+        });
+        Agent.agent.signifyNotifications.addPendingOperationToQueue(
+          pendingOperation
+        );
+      } else {
+        throw new Error(ConnectionService.FAILED_TO_RESOLVE_OOBI);
+      }
```

Comment:
What about if it is a multi-sig? Now, if the operation is still not done after 15 seconds, we don't throw  `ConnectionService.FAILED_TO_RESOLVE_OOBI` anymore